dwyl/time/stringToHash.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  stringToHash.swift
+//  dwyl
+//
+//  Created by Sohil Pandya on 22/06/2017.
+//  Copyright © 2017 dwyl. All rights reserved.
+//
+
+import Foundation
+
+func sha256(_ data: Data) -> Data? {
```

Comment:
@sohilpandya this looks good. we're going to need to write a _test_ to check for the output at some point _soon_ but for now this is fine. thanks! 👍